lib/wait-on.js

```diff
@@ -5,7 +5,7 @@ var Joi = require('@hapi/joi');
 var net = require('net');
 var Request = require('request');
 var Rx = require('rx');
-require('core-js/modules/es6.string.starts-with');
```

Comment:
If this is the only usage of core-js, I'd recommend using a polyfill for `startsWith` directly without core-js and then dropping the dependency on core-js. 
Here is a `startsWith` polyfill from MDN: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith#Polyfill